docs/gateway/b2.md

```diff
@@ -4,14 +4,18 @@ Minio Gateway adds Amazon S3 compatibility to Backblaze B2 Cloud Storage.
 ## Run Minio Gateway for Backblaze B2 Cloud Storage
 Please follow this [guide](https://www.backblaze.com/b2/docs/quick_account.html) to create an account on backblaze.com to obtain your access credentisals for B2 Cloud storage.
 
-### Using Binary
-Please download the test binary for gateway B2 support https://data.minio.io:10000/minio-b2/linux-amd64/minio-b2
-
+### Using Docker
 ```
-export MINIO_ACCESS_KEY=b2_accound_id
-export MINIO_SECRET_KEY=b2_application_key
-minio gateway b2
+docker run -p 9000:9000 --name azure-s3 \
+ -e "MINIO_ACCESS_KEY=b2_accound_id" \
+ -e "MINIO_SECRET_KEY=b2_application_key" \
+ minio/minio:edge gateway b2
 ```
+
+### Using Binary
+
+> WIP
```

Comment:
Instead of WIP we can probably remove this section till we make the release? or just say build from master?

Reply:
Yes we can.